linera-core/src/local_node.rs

```diff
@@ -220,8 +228,36 @@ where
                         return info;
                     }
                 }
-                result = self.handle_certificate(certificate.clone(), values).await;
+                result = self
+                    .handle_certificate(certificate.clone(), values, BTreeMap::new())
+                    .await;
+            } else if let Err(LocalNodeError::WorkerError(WorkerError::BlobsNotFound(blob_ids))) =
+                &result
+            {
```

Comment:
We need to handle the case where both certificate values and blobs are missing:

```suggestion
                    .await;
            }
            if let Err(LocalNodeError::WorkerError(WorkerError::BlobsNotFound(blob_ids))) =
                &result
            {
```

(This only works if in that case, the server _first_ returns the error about the certificate values, then about the blob. Otherwise it needs to be the other way round.)